samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/author/CalculatedQuestionGlobalVariableBean.java

```diff
@@ -0,0 +1,134 @@
+/**********************************************************************************
+ * $URL$
+ * $Id$
+ ***********************************************************************************
+ *
+ * Copyright (c) 2004, 2005, 2006, 2008, 2009 The Sakai Foundation
+ *
+ * Licensed under the Educational Community License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.opensource.org/licenses/ECL-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ **********************************************************************************/
+
+package org.sakaiproject.tool.assessment.ui.bean.author;
+
+import java.io.Serializable;
+
+/**
+ * CalculatedQuestionGlobalVariableBean stores the definition of a global variable for
+ * a calculated question
+ * @author jesusmmp
+ */
+public class CalculatedQuestionGlobalVariableBean implements Serializable, CalculatedQuestionAnswerIfc {
+
+    private static final long serialVersionUID = 1747088544228808857L;
+    private Long sequence;
+    private String name;
+    private String formula;
+    private boolean active;
+    private transient boolean validatedFormula;
+    private boolean fromui;
+
+    private static final String DEFAULT_FORMULA = "0";
+
+    public CalculatedQuestionGlobalVariableBean() {

```

Comment:
Do you think we can use Lombok here for sequence , name, text and formula? At least some @setter @getter

Just thinking loud again.

Reply:
Changed to lombok notation